<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd">
<suite name="Failed suite [Default suite]" guice-stage="DEVELOPMENT">
  <test thread-count="5" name="Default test(failed)">
    <classes>
      <class name="com.qa.alias.tests.TC005_without_vpn_verify_user_is_able_to_add_number_of_products_to_PDP_details_to_excel_with_sell_data2">
        <methods>
          <include name="verifyUserIsAbleToAddPDPDetailsToExcel"/>
          <include name="tearDown"/>
          <include name="startAppiumServer"/>
          <include name="openTheAppliaction"/>
          <include name="stopAppiumServer"/>
        </methods>
      </class> <!-- com.qa.alias.tests.TC005_without_vpn_verify_user_is_able_to_add_number_of_products_to_PDP_details_to_excel_with_sell_data2 -->
    </classes>
  </test> <!-- Default test(failed) -->
</suite> <!-- Failed suite [Default suite] -->
